website/docs/contribute.md

```diff
@@ -22,8 +22,9 @@ To submit a template:
     - **Author's Name** - Name to credit on the gallery card
     - **Link to template source** - Link to the template GitHub repo
     - **Tags** - One or more [tags](https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx) representing the template. Provide at least 1 tag for programming language used and at least 1 tag for Azure services integrated. Also tag the IaC provider (Bicep or Terraform). If you don't see a relevant tag for your template? Feel free to add one!
-2. Open a PR!
-3. If possible, add a link to the PR in your repo where you made your app `azd` compatible to the PR description. This will help us provide feedback on your template and speed up the review process. 
+2. If the template is microsoft authorized, we encourage to [publish the template to azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

Comment:
```suggestion
2. If the template is Microsoft authorized, we encourage to [publish the template to Azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

Reply:
What does Microsoft authorized mean? Is this one that is authored by a Microsoft employee? Or part of a Microsoft curated collection of templates?